src/rosdiscover/config.py

```diff
@@ -86,6 +95,7 @@ def from_dict(cls, dict_: t.Mapping[str, t.Any]) -> 'NodeSourceInfo':
             package_name=dict_['package'],
             node_name=dict_['node'],
             node_kind=kind,
+            entry_point=dict_['entry_point'],
```

Comment:
should we do `dict_.get("entry_point")` to avoid the need to specify entrypoints for normal nodes?